x-pack/plugins/screenshotting/server/browsers/chromium/templates/footer.html

```diff
@@ -0,0 +1,37 @@
+  }
```

Comment:
user's custom logo?

Reply:
I was thinking of that as a future enhancement. Perhaps we could change this to a handlebars template to make those values dynamic.